flow-typed/npm/vscode-debugprotocol_vx.x.x.js

```diff
@@ -0,0 +1,32 @@
+// flow-typed signature: 6b3104f896ee95abdc8475a4079566eb
+// flow-typed version: <<STUB>>/vscode-debugprotocol_vx.x.x/flow_v0.54.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'vscode-debugprotocol'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'vscode-debugprotocol' {
+  declare module.exports: any;
+}
+
+/**
+ * We include stubs for each file inside this npm package in case you need to
+ * require those files directly. Feel free to delete any files that aren't
+ * needed.
+ */
+declare module 'vscode-debugprotocol/lib/debugProtocol' {
+  declare module.exports: any;
+}
+
+// Filename aliases
+declare module 'vscode-debugprotocol/lib/debugProtocol.js' {
+  declare module.exports: $Exports<'vscode-debugprotocol/lib/debugProtocol'>;
+}
```

Comment:
This file doesn't seem very useful (it's mostly all `any`) & I can't find it in https://github.com/flowtype/flow-typed -- which should be the source of truth for any npm module types.

Reply:
This file was autogenerated using the CLI from https://github.com/flowtype/flow-typed because it's not there already. It is needed to satisfy flow checks.

Reply:
If the file doesn't add any value, you should just add it to the `ignore` section in `.flowconfig`.